index.js

```diff
@@ -55,7 +71,6 @@ module.exports = {
       }
     }],
     'linebreak-style': [2, 'unix'],
-    'max-len': [2, 80, 4],
```

Comment:
Why removed? I don't see this covered by prettier.

Reply:
It is covered with https://prettier.io/docs/en/options.html#print-width, which uses 80 characters as default.

However, there's some conflict between how prettier formats text and what ESLint approves of:

> Usually, Prettier takes care of following a maximum line length automatically. However, there are cases where Prettier can’t do anything, such as for long strings, regular expressions and comments. Those need to be split up by a human.
>
> If you’d like to enforce an even stricter maximum line length policy than Prettier can provide automatically, you can enable this rule. Just remember to keep `max-len`’s options and Prettier’s `printWidth` option in sync.
>
> Keep in mind that you might have to refactor code slightly if Prettier formats lines in a way that the `max-len` rule does not approve of.

https://github.com/prettier/eslint-config-prettier#max-len

Reply:
That's fair. I'm fine with exceptional cases going over the limit.